packages/web/src/components/link/ExternalLink.tsx

```diff
@@ -12,10 +12,17 @@ import { Link, LinkProps } from './Link'
 
 type ExternalLinkProps = LinkProps & {
   source?: 'profile page' | 'track page' | 'collection page'
+  showWarningIfNotAllowed?: boolean
```

Comment:
super nit: this name is kinda confusing - am I not allowed to go to the page? maybe `treatAsInternal` with the logic inverted?

Reply:
I updated this to `showWarning` because I think that's easier to understand to the reader than `treatAsInternal` - the ifNotAllowed thing is confusing.

Reply:
maybe `isInternal`? `showWarning` makes me think it will _always_ show the warning. Or `ignoreWarning`?

Reply:
i like `ignoreWarning`!
 `isInternal` is so not obvious to me. i spent a while looking for the right component to use here